go.mod

```diff
@@ -87,49 +87,49 @@ require (
 
 require (
 	github.com/distribution/reference v0.6.0 // indirect
-	github.com/open-telemetry/opentelemetry-collector-contrib/connector/spanmetricsconnector v0.109.0
```

Comment:
This update is not related to this pull request, right? Let's do this separately if possible.

Reply:
Yeah, you're right but I needed to update the `filestorage` extension to the latest version, so I updated all the OTeL dependencies to the latest version as well to maintain consistency. Otherwise, some would dependencies be at `0.109.0` and others at `0.110.0`.

Reply:
Oh, OK then.